src/sidebar/pin/tree/NoPinnedItemTreeItem.ts

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
Do we need the empty space here?

Reply:
We should not have it indeed. I will fix it